webapp/security/alembic/versions/a88c140f4ed1_cve_migration.py

```diff
@@ -99,7 +99,7 @@ def upgrade():
         "cve", sa.Column("last_updated_date", sa.DateTime(), nullable=True)
     )
     op.add_column("cve", sa.Column("mitigation", sa.String(), nullable=True))
-    op.add_column("cve", sa.Column("notes", sa.String(), nullable=True))
+    op.add_column("cve", sa.Column("notes", sa.JSON(), nullable=True))
```

Comment:
Why JSON? 'Cos you want to store them line-by-line?

Reply:
So the current ubuntu tracker is storing them as a pair of author/comment. I think it's useful to keep it with this data structure for display, but since we don't need to query them by author or comment, we don't need to create tables, so I stored them as a JSON.

Reply:
Ah yes. Nice.